code/game/gamemodes/miniantags/morph/morph.dm

```diff
@@ -0,0 +1,179 @@
+#define MORPH_COOLDOWN 50
+
+/mob/living/simple_animal/hostile/morph
+	name = "morph"
+	real_name = "morph"
+	desc = "A revolting, pulsating pile of flesh."
+	speak_emote = list("gurgles")
+	emote_hear = list("gurgles")
+	icon = 'icons/mob/animal.dmi'
+	icon_state = "morph"
+	icon_living = "morph"
+	icon_dead = "morph_dead"
+	speed = 2
+	a_intent = "harm"
+	stop_automated_movement = 1
+	status_flags = CANPUSH
+	pass_flags = PASSTABLE
+	ventcrawler = 2
+
+	min_oxy = 0
+	max_oxy = 0
+	min_tox = 0
+	max_tox = 0
+	min_co2 = 0
+	max_co2 = 0
+	min_n2 = 0
+	max_n2 = 0
+
+	minbodytemp = 0
+	maxHealth = 150
+	health = 150
+	environment_smash = 1
+	melee_damage_lower = 20
+	melee_damage_upper = 20
+	see_in_dark = 8
+	see_invisible = SEE_INVISIBLE_MINIMUM
+	idle_vision_range = 1 // Only attack when target is close
+	wander = 0
+	attacktext = "glomps"
+	attack_sound = 'sound/effects/blobattack.ogg'
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/slab
+
+	var/morphed = 0
+	var/atom/movable/form = null
+	var/morph_time = 0
+
+	var/playstyle_string = "<b><font size=3 color='red'>You are a morph,</font> an abomination of science created primarily with changeling cells. \
+							You may take the form of anything nearby by shift-clicking it. This process will alert any nearby \
+							observers, and can only be performed once every five seconds. While morphed, you move faster, but do \
+							less damage. In addition, anyone within three tiles will note an uncanny wrongness if examining you. \
+							You can attack any item or dead creature to consume it - creatures will fully restore your health. \
```

Comment:
Ah, this might have been the line that gave me misconceptions - I should probably make this line more accurate